src/ctia/auth/jwt.clj

```diff
@@ -10,13 +10,19 @@
     [auth :as auth :refer [IIdentity]]
     [properties :as prop]]))
 
-(def entity-root-scope
-  (get-in @prop/properties [:ctia :auth :entities :scope]
-          "private-intel"))
+(def entity-root-scope (atom nil))
```

Comment:
I would much prefer if we change the `def` to `defn` instead of creating an atom just for that